app/scripts/views/sign_in.js

```diff
@@ -18,6 +17,7 @@ define(function (require, exports, module) {
   const MigrationMixin = require('views/mixins/migration-mixin');
   const PasswordMixin = require('views/mixins/password-mixin');
   const PasswordResetMixin = require('views/mixins/password-reset-mixin');
+  const { preventDefaultThen, t } = require('views/base');
```

Comment:
Huh... Odd... I wonder why the linter wasn't complaining about an unused variable. I'll have a